code/modules/mob/new_player/new_player.dm

```diff
@@ -319,8 +319,9 @@
 		if(prob(10)) // 10% of those have a good mut.
 			H.dna.GiveRandomSE(notflags = GENE_UNNATURAL,genetype = GENETYPE_GOOD)
 
-/mob/new_player/proc/DiseaseCarrierCheck(var/mob/living/carbon/human/H)
+/mob/new_player/proc/DiseaseCarrierCheck(var/mob/living/carbon/human/H)  
 	// 5% of players are joining the station with some minor disease
+	/*  Uncomment this if somebody wants roundstart viruses again in the future.
```

Comment:
You do realize that with this comment you are going to be perpetuating the same mistake that you suffered from